src/modm/platform/fault/minimal/module.lb

```diff
@@ -15,7 +15,7 @@ import os
 
 def init(module):
     module.parent = "platform"
-    module.name = "fault.cortex"
+    module.name = "fault.minimal"
```

Comment:
Is there a reason for renaming this from _cortex_ to _minimal_?
This module is completely ARM Cortex-M specific.

Reply:
So is `fault.coredump`. I'd prefer to not encode the device id into the module name, since they themselves decide which devices to support. Maybe with a different platform the `fault.minimal` module could be implemented differently (Cortex-A, RiscV, Xtensa) but with the same functionality.

Reply:
Btw, this is how all of `modm:platform:**` modules work, the `modm:platform:gpio` module for AVRs implements the same functionality as the one for STM32, but they don't conflict, since only one is enabled per platform.